assets/tests/components/searchResults.test.tsx

```diff
@@ -492,7 +495,7 @@ describe("SearchResults", () => {
     const testDispatch = jest.fn()
     const wrapper = mount(
       <StateDispatchProvider state={state} dispatch={testDispatch}>
-        <SearchResults vehicles={[]} />
+        <SearchResults vehicles={[]} routes={[]} />
```

Comment:
I think these props in the tests are obsolete now.

Reply:
Do you know a way to force recompilation of all tests? Webpack doesn't seem to consistently catch breaking changes in tests--meaning, really, I don't understand the exact conditions. But I'd rather just be able to start fresh when I want to.